"HomeActivity" crash after the refactor to Kotlin (2f341ad)

With the new commit converting `HomeActivity.java` to `HomeActivity.kt` , assignment of `HomeActivity.layoutManager` to `HomeActivity.grid` was missed which caused the following crash.
```
java.lang.RuntimeException: Unable to start activity ComponentInfo{io.plaidapp/io.plaidapp.ui.HomeActivity}: kotlin.TypeCastException: null cannot be cast to non-null type androidx.recyclerview.widget.GridLayoutManager
        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3121)
        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3257)
        at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:81)
        at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:135)
        at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:95)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1935)
        at android.os.Handler.dispatchMessage(Handler.java:107)
        at android.os.Looper.loop(Looper.java:214)
        at android.app.ActivityThread.main(ActivityThread.java:7116)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:492)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:925)
     Caused by: kotlin.TypeCastException: null cannot be cast to non-null type androidx.recyclerview.widget.GridLayoutManager
        at io.plaidapp.ui.HomeActivity.setupGrid(HomeActivity.kt:393)
        at io.plaidapp.ui.HomeActivity.onCreate(HomeActivity.kt:290)
        at android.app.Activity.performCreate(Activity.java:7698)
        at android.app.Activity.performCreate(Activity.java:7687)
        at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1299)
        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3096)
        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3257) 
        at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:81) 
        at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:135) 
        at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:95) 
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1935) 
        at android.os.Handler.dispatchMessage(Handler.java:107) 
        at android.os.Looper.loop(Looper.java:214) 
        at android.app.ActivityThread.main(ActivityThread.java:7116) 
        at java.lang.reflect.Method.invoke(Native Method) 
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:492) 
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:925) 
```

--- a/app/src/main/java/io/plaidapp/ui/HomeActivity.kt
+++ b/app/src/main/java/io/plaidapp/ui/HomeActivity.kt
@@ -104,7 +104,7 @@ class HomeActivity : AppCompatActivity() {
 
     private var columns = 0
     private var filtersAdapter = FilterAdapter()
-    private lateinit var layoutManager: GridLayoutManager
+    private lateinit var gridLayoutManager: GridLayoutManager
     private lateinit var drawer: DrawerLayout
 
     private lateinit var toolbar: Toolbar
@@ -188,8 +188,8 @@ class HomeActivity : AppCompatActivity() {
             // toolbar is laid out in front of the grid but when we scroll, we lower it's elevation
             // to allow the content to pass in front (and reset when scrolled to top of the grid)
             if (newState == RecyclerView.SCROLL_STATE_IDLE &&
-                layoutManager.findFirstVisibleItemPosition() == 0 &&
-                layoutManager.findViewByPosition(0)!!.top == grid.paddingTop &&
+                gridLayoutManager.findFirstVisibleItemPosition() == 0 &&
+                gridLayoutManager.findViewByPosition(0)!!.top == grid.paddingTop &&
                 toolbar.translationZ != 0f
             ) {
                 // at top, reset elevation
@@ -348,7 +348,7 @@ class HomeActivity : AppCompatActivity() {
     }
 
     private fun setupGrid() {
-        layoutManager = GridLayoutManager(this@HomeActivity, columns).apply {
+        gridLayoutManager = GridLayoutManager(this@HomeActivity, columns).apply {
             spanSizeLookup = object : GridLayoutManager.SpanSizeLookup() {
                 override fun getSpanSize(position: Int): Int {
                     return feedAdapter.getItemColumnSpan(position)
@@ -356,7 +356,7 @@ class HomeActivity : AppCompatActivity() {
             }
         }
         val infiniteScrollListener =
-            object : InfiniteScrollListener(layoutManager) {
+            object : InfiniteScrollListener(gridLayoutManager) {
                 override fun onLoadMore() {
                     viewModel.loadData()
                 }
@@ -376,6 +376,7 @@ class HomeActivity : AppCompatActivity() {
         )
 
         with(grid) {
+            layoutManager = gridLayoutManager
             adapter = feedAdapter
             addOnScrollListener(toolbarElevation)
             addOnScrollListener(infiniteScrollListener)
@@ -389,8 +390,6 @@ class HomeActivity : AppCompatActivity() {
             itemAnimator = HomeGridItemAnimator()
             addOnScrollListener(shotPreloader)
         }
-
-        layoutManager = grid.layoutManager as GridLayoutManager
     }
 
     private fun handleDrawerInsets(insets: WindowInsets) {

